lib/sycamore/sycamore/query/client.py

```diff
@@ -245,18 +251,33 @@ def default_text_embedder():
         return SentenceTransformerEmbedder(batch_size=100, model_name="sentence-transformers/all-MiniLM-L6-v2")
 
     @staticmethod
-    def _get_default_context(s3_cache_path, os_client_args, sycamore_exec_mode) -> Context:
+    def _get_default_context(
+        llm_cache_dir: Optional[str],
+        os_client_args: Optional[dict],
+        sycamore_exec_mode: ExecMode,
+        llm: Optional[Union[str, LLM]],
```

Comment:
this is fine for now, but using a bigboy llm for the record level operators will probably break

Reply:
Break or just be slow and expensive?